packages/formstr-app/src/components/MyForms/index.js

```diff
@@ -60,10 +60,10 @@ const MyForms = () => {
       .sort((a, b) => {
         return new Date(b.createdAt) - new Date(a.createdAt);
       });
-    if (!formDrafts) {
+    if (!formDrafts && drafts.length !== 0) {
```

Comment:
`!formDrafts?.length === 0`

Reply:
this component should not dependent on multiple variables. It should be only accepting drafts as a prop or initialise it as a state and only use that variable

Reply:
agreed out of scope for this PR though